scripts/verify_security.py

```
@@ -0,0 +1,122 @@
+    proc = subprocess.Popen(
+        [sys.executable, "api_server.py"],
+        env=env,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+    )
+
+    # Wait for server to start
+    time.sleep(2)
+
+    try:
+        response = requests.get(f"http://localhost:{port}/health")
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)